tests/store/artifact/test_databricks_artifact_repo.py

```diff
@@ -1106,6 +1106,35 @@ def mock_upload_to_cloud(
             assert f.read() == "file1"
 
 
+def test_artifact_logging_chunks_upload_list(databricks_artifact_repo, tmpdir):
+    """
+    Verifies that write credentials are fetched in chunks rather than all at once.
+    """
+    src_dir = os.path.join(str(tmpdir), "src")
+    os.makedirs(src_dir)
+    for i in range(10):
+        src_file_path = os.path.join(src_dir, f"file_{i}.txt")
+        with open(src_file_path, "w") as f:
+            f.write(f"file{i}")
+    dst_dir = os.path.join(str(tmpdir), "dst")
+    os.makedirs(dst_dir)
```

Comment:
```suggestion
    src_dir = tmp_path.joinpath("src")
    src_dir.mkdir()
    for i in range(10):
        src_dir.joinpath("f"file_{i}.txt"").write_text(f"file{i}")
    dst_dir = tmp_path.joinpath("dst")
    dst_dir.mkdir()
```

Can we use `tmp_path` instead of `tmpdir`?